src/System.Data.SqlClient/src/System/Data/SqlClient/SqlCommand.cs

```diff
@@ -3261,6 +3346,14 @@ internal void CancelIgnoreFailure()
             }
         }
 
+        private void NotifyDependency()
+        {
+            if (_sqlDep != null)
+            {
+                _sqlDep.StartTimer(Notification);
```

Comment:
Seems like SqlDependency.Invalidate() handles that. There's no StopTimer() method in the Framework code.